go/libraries/doltcore/sqle/sglobal/session_global.go

```diff
@@ -0,0 +1,49 @@
+package sglobal
+
+import (
+	"sync"
+)
+
+type RefInMemStore interface {
+	GetRefMemStore(refhash string) *refMemStore
+	SetRefMemStore(refhash string, store *refMemStore)
+}
```

Comment:
Don't try to expand this interface beyond the AI tracker for now. Just a single getter methods for the AI tracker on this interface, GetAutoIncrementTracker(ref.WorkingSetRef). You don't need a setter: just return a new AI tracker as needed when one isn't registered for the ref given

Call the interface GlobalState for now